crypto-java-port.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
```

Comment:
This should be removed before merge.
Happy for this content to exist on the PR description.

Reply:
Agreed; that was always my intention

Reply:
deleted